README.md

```diff
@@ -5,16 +5,32 @@ tutorial: https://github.com/s-matyukevich/raspberry-pi-os.git
 
 ## Chainloader
 
-```make flash``` copies the chainloader onto the boot partition of the sd card (may not work, depends on system
-configuration). The chainloader uses the STATUS led ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)
-and the POWER led ![#f02015](https://via.placeholder.com/15/f02015/000000?text=+) to indicate its current status. Read
-as follows:
+To make mounting of the SD card in the makefile work it is currently required to have set up the SD card in `/etc/fstab` to mount in the mnt directory at the project directory root level.  
+An example config line would be:
 
-* ![#444444](https://via.placeholder.com/15/444444/000000?text=+)![#444444](https://via.placeholder.com/15/444444/000000?text=+)
+```
+# /dev/mmcblk0p1
+UUID=7616-4FD8    /home/kjell/git/rose/mnt    vfat   auto,nofail,noatime,user,rw    0 2
+```
+
+`make flash` mounts the SD card, copies the chainloader onto its boot partition and then unmounts it again.
+
+The chainloader uses the **STATUS** led ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)
+and the **POWER** led ![#f02015](https://via.placeholder.com/15/f02015/000000?text=+) to indicate its current status.  
+Read as follows:
+
+- ![#444444](https://via.placeholder.com/15/444444/000000?text=+)![#444444](https://via.placeholder.com/15/444444/000000?text=+)
   ---> Raspberry is off
-* ![#444444](https://via.placeholder.com/15/444444/000000?text=+)![#f02015](https://via.placeholder.com/15/f02015/000000?text=+)
+- ![#444444](https://via.placeholder.com/15/444444/000000?text=+)![#f02015](https://via.placeholder.com/15/f02015/000000?text=+)
   ---> Chainloader makes space. Shouldn't take long. If you see this, your chainloader may be stuck
-* ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#444444](https://via.placeholder.com/15/444444/000000?text=+)
+- ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#444444](https://via.placeholder.com/15/444444/000000?text=+)
   ---> Chainloader is waiting for 32 bit kernel size
-* ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#f02015](https://via.placeholder.com/15/f02015/000000?text=+)
-  ---> Chainloader is waiting for kernel
\ No newline at end of file
+- ![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#f02015](https://via.placeholder.com/15/f02015/000000?text=+)
+  ---> Chainloader is waiting for kernel
+
+## Loading the system
+
+Once the Chainloader is waiting for the kernel size (![#80f015](https://via.placeholder.com/15/80f015/000000?text=+)![#f02015](https://via.placeholder.com/15/444444/000000?text=+)) you can build and load the image onto the Raspberry Pi with `make send`.
+
+
+If you already loaded and booted the system you can restart the Pi, to start the chainloader again and rebuild + send the new image, with `make resend`.
```

Comment:
It may be important to add that `make resend` will freeze the raspberry if the raspberry was waiting for `make send`

Reply:
Oh, yeah. Also should add that you have to hard reset and `make send` again if the pi freezes

Reply:
Technically, we could have the chainloader filter out the special character we use to reboot the raspberry.
We could just always send one character before we send the actual size. Wouldn't actually be hard

Reply:
this would need to be handled in a separate PR